src/mmw/apps/modeling/tasks.py

```diff
@@ -213,15 +213,19 @@ def format_quality(model_output):
                 'Total Phosphorus']
     codes = ['tss', 'tn', 'tp']
 
-    def fn(input):
-        measure, code = input
-        return {
-            'measure': measure,
-            'load': model_output['modified'][code] * KG_PER_POUND,
-            'runoff': model_output['modified']['runoff']  # Already CM
-        }
-
-    return map(fn, zip(measures, codes))
+    quality = {}
+    for key in model_output:
+        def fn(input):
```

Comment:
Since we're already updating this code here, I wonder if it would make sense to give this function a more descriptive name so it's easy to tell at a glance why it's reformatting the quality data.